src/cryptoadvance/specter/commands/psbt_creator.py

```diff
@@ -138,29 +138,31 @@ def paymentinfo_from_ui(cls, specter, wallet, request_form):
         """calculates the correct format needed by wallet.createpsbt() out of a request-form
```

Comment:
Can you have a look in line 31. Is that description of how the `request_form` look like? Can you update that? 
@stepansnigirev might be happy as he has created his exfund extension using this API:
https://github.com/stepansnigirev/specterext-exfund/blob/master/src/cryptoadvance/specterext/exfund/controller.py#L68

Reply:
I tried with json, but it was giving me troubles with liquid assets, so I switched to the request_form.
I should probably try with json again, add a few tests and report back if something doesn't work especially on Liquid. I hope I can get to it on Saturday.

Reply:
@moneymanolis took a screenshot from the browser. This is how it looks like in this PR: 
![image](https://user-images.githubusercontent.com/117085/181006260-77757276-8e74-4116-8b24-aaa3bb274a3d.png)

Reply:
@relativisticelectron, if this is fine how I updated it in this commit https://github.com/cryptoadvance/specter-desktop/pull/1782/commits/033baa6007638e72274834c19fac2697ea4f85b3 we can resolve this.